tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb

```diff
@@ -0,0 +1,459 @@
+    "\n",
+    "coco_metric = CocoEval(EVAL_DATASET_ANNOTATION_FILE)\n",
+    "for batch_idx, (images, targets) in enumerate(data_loader):\n",
+    "    # Run inference on the batch\n",
+    "    images = list(image.to(device) for image in images)\n",
+    "    with torch.no_grad():\n",
+    "        outputs = model(images)\n",
+    "\n",
+    "    coco_metric.add_batch_detections(outputs, targets)\n",
+    "    if (batch_idx+1) % 10 == 0:\n",
+    "        print(f'processed {(batch_idx+1)*data_loader.batch_size} images')\n",
+    "\n",
+    "# Print mAP results\n",
+    "print(\"mAP: {:.4f}\".format(coco_metric.result()[0]))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "015e760b-6555-45b4-aaf9-500e974c1d86",
+   "metadata": {},
+   "source": [
+    "### Extract model to be quantized\n",
+    "\n",
+    "Extract the float model's backcone and head, and construct a torch model the only runs them"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01e90967-594b-480f-b2e6-45e2c9ce9cee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class SDD4Quant(torch.nn.Module):\n",
+    "    def __init__(self, in_sdd, *args, **kwargs):\n",
+    "        super().__init__(*args, **kwargs)\n",
+    "        self.add_module(\"base\", in_sdd)\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        features = self.base.backbone(x)\n",
+    "\n",
+    "        features = list(features.values())\n",
+    "\n",
+    "        # compute the ssd heads outputs using the features\n",
+    "        head_outputs = self.base.head(features)\n",
+    "        return head_outputs\n",
+    "\n",
+    "\n",
+    "model4quant = SDD4Quant(model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08fb59fd-3877-45b4-8529-7f9edb687c69",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Extract preproecss and postprocess\n",
+    "\n",
+    "Extract the preprocess and postprocess functions from the float model object, and construct separate preprocess and postprocess functions for the representative dataset and evaluation code\n",
+    "\n",
+    "\n",
+    "Note: the MCT output model flattens the float model output data structure to a list, so the PostProcess manually rebuilds it as the original data structure (a dictionary)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ff336c30-56c9-4de8-9c42-6462ddb8d2c0",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def preprocess(image, targets):\n",
+    "    # need to save the original image sizes before resize for the postprocess part\n",
+    "    targets = {'gt': targets, 'img_size': list(image.size[::-1])}\n",
+    "    image = model.transform([torchvision.transforms.ToTensor()(image)])[0].tensors[0, ...]\n",
+    "    return image, targets\n",
+    "\n",
+    "\n",
+    "class PostProcess:\n",
```

Comment:
Either convert it to a function or change the comment above: ""Extract the preprocess and postprocess functions "

Reply:
I wrote it as a class so I don't have to create the self.features list of tensors every time the postprocess is called